src/app/Models/PruningType.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace App\Models;
+
+use App\Core\Database;
+use PDO;
+use PDOException;
+
+class PruningType
+{
+    private $id;
+    private $name;
+    private $description;
```

Comment:
Pots afegir el tipus de cada un de les variables ^^
```suggestion
    private int $id;
    private string $name;
    private string $description;
```